specification/requirements.adoc

```diff
@@ -1,5 +1,167 @@
 [[requirements]]
 == Requirements
 
-Hardware (platform and device) and software (host and guest) requirements to
-support CoVE-IO.
+In order to extend TVM TCBs with external and untrusted devices, the host
+platform and software stack, together with assigned devices, must meet a
+specific set of requirements.
+
+=== Device
+
+CoVE-IO compliant implementations support extending TVMs TCBs with devices if
+and only if they meet the following requirements:
+
+==== PCI
+
+A CoVE-IO supported device must be a PCIe device with optional SR-IOV
```

Comment:
> must be a PCIe device

I know CXL.mem is too far. But do we want to support CXL.io device?

The charter says: "The recommendations will refer to ... and CXL proposed standards."

Reply:
CXL.io seems reasonable indeed. Added.